package-lock.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "integreat-compass",
+  "lockfileVersion": 2,
+  "requires": true,
+  "packages": {
+    "": {
+      "devDependencies": {
+        "prettier": "^2.8.4"
```

Comment:
Could you run `npm install` one last time to update the version here as well?

Reply:
Sorry about all those details :(